core/src/main/java/com/sequenceiq/distrox/v1/distrox/controller/DistroXV1Controller.java

```diff
@@ -55,95 +56,260 @@
     private DistroXMaintenanceModeV1ToMainenanceModeV4Converter maintenanceModeConverter;
 
     @Override
-    public StackViewV4Responses list(String environmentName) {
-        return stackOperation.listByEnvironmentName(workspaceService.getForCurrentUser().getId(), environmentName, StackType.WORKLOAD);
+    public StackViewV4Responses list(String environmentName, String environmentCrn) {
```

Comment:
should fail if both are not null (or if they are not the same, but the prior is more simple)

Reply:
no because then we dont filter for environment